chalice/deploy/planner.py

```diff
@@ -1252,8 +1252,14 @@ def _plan_restapi(self, resource):
                     '/endpointConfiguration/types/%s' % (
                         '{rest_api[endpointConfiguration][types][0]}'),
                     ['rest_api']),
-                'value': resource.endpoint_type}
-            )
+                'value': resource.endpoint.type
+            })
+            if resource.endpoint.vpce_ids:
```

Comment:
You'll have to create a new patch operation for each VPCE, otherwise you'll get an error in botocore that the `value` has to be a string type but a list of strings was received instead.  This happens when you specify this config value as a list, e.g. `      "api_gateway_vpce_endpoint": ["vpce-abcdefg"],`

Reply:
Yes, completely related to the missing fact of possibility to attach to multiple VPC endpoints.